docs/api.md

```diff
@@ -350,6 +350,66 @@ class ResetPasswordPage extends React.Component {
 }
 ```
 
+#### ChangePasswordForm
+
+Renders a change password form. The parameter `spToken` is required in order for the token to be validated.
+
+```html
+<ChangePasswordForm spToken={this.props.location.query.sptoken} />
+```
+
+Customize the form by providing your own markup.
+
+```html
+<ChangePasswordForm>
```

Comment:
To avoid missing to add the `spToken` parameter when copying this, I think it would be nice to include it in this example too:

```html
<ChangePasswordForm spToken={this.props.location.query.sptoken}>
```